File: back-end/GeekSpot.Domain/Entities/AjudaTopico.cs
using System.ComponentModel.DataAnnotations;
using static GeekSpot.Utils.Biblioteca;

namespace GeekSpot.Domain.Entities
{
    public class AjudaTopico
    {
        [Key]
        public int AjudaTopicoId { get; set; }
        public string? Titulo { get; set; } = null;
        public string? Descricao { get; set; } = null;
        public bool IsAtivo { get; set; } = true;
        public DateTime? DataRegistro { get; set; } = HorarioBrasilia();
    }
}

File: front-end/utils/types/ajuda.topico.ts
export default interface iAjudaTopico {
    ajudaTopicoId: number;
    titulo: string;
    descricao:string;
    isAtivo: boolean;
    dataRegistro: Date;

    erro: boolean;
    codigoErro: number;
    mensagemErro: string | null;
}


